src/stacktraces.cpp

```diff
@@ -686,32 +686,34 @@ crash_info GetCrashInfoFromException(const std::exception_ptr& e)
     std::string type;
     std::string what;
 
+    auto getExceptionType = [&]() -> std::string {
+        auto type = abi::__cxa_current_exception_type();
+        if (type && (strlen(type->name()) > 0)) {
+            return DemangleSymbol(type->name());
+        }
+        return std::string("<unknown>");
```

Comment:
nit, but now with the explicit return type we could drop the explicit constructor call here imo
```suggestion
        return "<unknown>";
```